setup.py

```diff
@@ -21,6 +21,7 @@
       packages=['mwdblib', 'mwdblib.cli', 'mwdblib.cli.formatters'],
       package_data={"mwdblib": ["*.pyi", "py.typed"]},
       url="https://github.com/CERT-Polska/mwdblib",
+      license="MIT",
```

Comment:
Is it OK? I see that this argument is a bit... controversial and used only if you can't choose license from existing classifiers
https://packaging.python.org/specifications/core-metadata/#license

```
Text indicating the license covering the distribution where the license is not a selection from the “License” Trove classifiers. See “Classifier” below. This field may also be used to specify a particular version of a license which is named via the Classifier field, or to indicate a variation or exception to such a license.
```

https://github.com/pypa/packaging.python.org/issues/95

Reply:
oh, I thought that those were independent. :+1: